technical_analysis/trend.py

```diff
@@ -24,6 +24,8 @@ def adx(l_args, s_ticker, s_interval, df_stock):
             print(f"The following args couldn't be interpreted: {l_unknown_args}\n")
             return
 
+        plt.figure()
```

Comment:
I would put this straight above both plt.subplot(211), to be more contained in that area of code